apps/web/modules/ui/components/alert/stories.tsx

```
@@ -1,49 +1,250 @@
+  args: {
+    variant: "default",
+    title: "Alert Title",
+    description: "This is an important notification.",
+    showIcon: true,
+    showButton: true,
+    actionButtonText: "Learn more",
+  },
+};
 
+// Error variant
 export const Error: Story = {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)